code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -308,46 +308,55 @@ proc/issyndicate(mob/living/M as mob)
 		to_chat(world, "<B>[syndicate_name()] operatives destroyed [station_name()] but did not leave the area in time and got caught in the explosion.</B> Next time, don't lose the disk!")
 
 	else if(!disk_rescued && !station_was_nuked && nuke_off_station && !syndies_didnt_escape)
 		feedback_set_details("round_end_result","nuclear halfwin - blew wrong station")
+		GLOB.gamemode_end_state = "NUKEOPS-WRONG-STATION"
 		to_chat(world, "<FONT size = 3><B>Crew Minor Victory</B></FONT>")
 		to_chat(world, "<B>[syndicate_name()] operatives secured the authentication disk but blew up something that wasn't [station_name()].</B> Next time, don't lose the disk!")
 
 	else if(!disk_rescued && !station_was_nuked && nuke_off_station && syndies_didnt_escape)
 		feedback_set_details("round_end_result","nuclear halfwin - blew wrong station - did not evacuate in time")
+		GLOB.gamemode_end_state = "NUKEOPS-WRONG-STATION-NUKIES-DIED"
 		to_chat(world, "<FONT size = 3><B>[syndicate_name()] operatives have earned Darwin Award!</B></FONT>")
 		to_chat(world, "<B>[syndicate_name()] operatives blew up something that wasn't [station_name()] and got caught in the explosion.</B> Next time, don't lose the disk!")
 
 	else if(disk_rescued && is_operatives_are_dead())
 		feedback_set_details("round_end_result","nuclear loss - evacuation - disk secured - syndi team dead")
+		GLOB.gamemode_end_state = "NUKEOPS-CREW-MAJOR-OPSDEAD"
 		to_chat(world, "<FONT size = 3><B>Crew Major Victory!</B></FONT>")
 		to_chat(world, "<B>The Research Staff has saved the disc and killed the [syndicate_name()] Operatives</B>")
 
 	else if(disk_rescued)
 		feedback_set_details("round_end_result","nuclear loss - evacuation - disk secured")
+		GLOB.gamemode_end_state = "NUKEOPS-CREW-MAJOR"
 		to_chat(world, "<FONT size = 3><B>Crew Major Victory</B></FONT>")
 		to_chat(world, "<B>The Research Staff has saved the disc and stopped the [syndicate_name()] Operatives!</B>")
 
 	else if(!disk_rescued && is_operatives_are_dead())
 		feedback_set_details("round_end_result","nuclear loss - evacuation - disk not secured")
+		GLOB.gamemode_end_state = "NUKEOPS-CREW-MINOR-NODISK"
 		to_chat(world, "<FONT size = 3><B>Syndicate Minor Victory!</B></FONT>")
 		to_chat(world, "<B>The Research Staff failed to secure the authentication disk but did manage to kill most of the [syndicate_name()] Operatives!</B>")
 
 	else if(!disk_rescued && crew_evacuated)
 		feedback_set_details("round_end_result","nuclear halfwin - detonation averted")
+		GLOB.gamemode_end_state = "NUKEOPS-NONUKE"
 		to_chat(world, "<FONT size = 3><B>Syndicate Minor Victory!</B></FONT>")
 		to_chat(world, "<B>[syndicate_name()] operatives recovered the abandoned authentication disk but detonation of [station_name()] was averted.</B> Next time, don't lose the disk!")
 
 	else if(!disk_rescued && !crew_evacuated)
 		feedback_set_details("round_end_result","nuclear halfwin - interrupted")
+		GLOB.gamemode_end_state = "NUKEOPS-FUCK"

```

Comment:
oops

Reply:
Do you want this to be something else? Its that so its very clear if the server breaks

Reply:
No I'm just chuckling at this, it's 100% OK